docs/operators/flatten.md

```diff
@@ -0,0 +1,99 @@
+## `flatten` operator
+
+The `flatten` operator flattens a field by moving its children up to the same level as the field.
+
+### Configuration Fields
+
+| Field      | Default          | Description                                                                                                                                                                                                                              |
+| ---        | ---              | ---                                                                                                                                                                                                                                      |
+| `id`       | `flatten`    | A unique identifier for the operator                                                                                                                                                                                                     |
+| `output`   | Next in pipeline | The connected operator(s) that will receive all outbound entries                                                                                                                                                                         |
+| `field`      | required       | The [field](/docs/types/field.md) to be flattened.                                                       |
+| `on_error` | `send`           | The behavior of the operator if it encounters an error. See [on_error](/docs/types/on_error.md)                                                                                                                                          |
+| `if`       |                  | An [expression](/docs/types/expression.md) that, when set, will be evaluated to determine whether this operator should be used for the given entry. This allows you to do easy conditional parsing without branching logic with routers. |
+
+Example usage:
+
+<hr>
+Flatten an object to the base of the body
+<br>
+<br>
+
+```yaml
+- type: flatten
+    field: key1
+```
+
+<table>
+<tr><td> Input Body </td> <td> Output Body </td></tr>
```

Comment:
Even though this operator only works on the body, let's use the same example format as the others (showing the entire entry).